module/spl/spl-proc.c

```diff
@@ -234,6 +234,8 @@ taskq_seq_show_headers(struct seq_file *f)
 #define LHEAD_ACTIVE	4
 #define LHEAD_SIZE	5
 
+#define	MAX_SHOW_TASK	256
```

Comment:
Perhaps this should be a tunable.  Often when I'm looking at the taskq kstat I'm debugging an issue and I want to make sure I'm seeing everything.  In these case it's entirely acceptable to stall the system to get the needed debugging.